cves/CVE-2013-4444.yml

```diff
@@ -109,11 +103,7 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer: |
-    The fix didn't involve unit testing, rather it was a "code cleanup" fix as
-    it required deletion of code. I think that not having unit tests
-    makes sense in this case as this vulnerability is an edge case that needs
-    limited conditions such as an outdated java.io.File in order to happen.
+  answer: Looking through the unit tests, I do not see any for file uploading.
```

Comment:
I think you should mention that the fix was code cleanup as the person who wrote this before did.